### PR TITLE
[GH-#118] Server timeout when checking peers_nonce

### PR DESCRIPTION
**Pull request by HristianHristov**
Resolves GH-#118

### CODE REVIEW

File: apps/aecore/lib/aecore/peers/worker.ex [hunk 1]
```diff
@@ -66,44 +62,52 @@ defmodule Aecore.Peers.Worker do
...
     {:ok, initial_peers}
   end
 
-def handle_call({:add_peer,uri}, _from, %{peers: peers, nonce: own_nonce} = state) do
+def handle_call({:add_peer,uri}, _from, peers) do
     case(Client.get_info(uri)) do
       {:ok, info} ->
-        case own_nonce == info.peer_nonce do
-          false ->  
+        case Aecore.Peers.Worker.get_peer_nonce() == info.peer_nonce do
```
Comment (d-velev) [inline apps/aecore/lib/aecore/peers/worker.ex]: we don't have to call the function here, just use @peer_nonce

File: apps/aecore/lib/aecore/peers/worker.ex [hunk 2]
```diff
@@ -10,21 +10,23 @@ defmodule Aecore.Peers.Worker do
   alias Aecore.Utils.Blockchain.BlockValidation
   alias Aehttpclient.Client, as: HttpClient
   alias Aecore.Utils.Serialization
+  alias Aecore.Peers.Scheduler, as: Scheduler
```
Comment (d-velev) [inline apps/aecore/lib/aecore/peers/worker.ex]: we don't need this

File: apps/aehttpserver/web/controllers/info_controller.ex [hunk 3]
```diff
@@ -6,6 +6,7 @@ defmodule Aehttpserver.InfoController do
   alias Aecore.Utils.Blockchain.BlockValidation
   alias Aecore.Keys.Worker, as: Keys
   alias Aecore.Peers.Worker, as: Peers
+  alias Aecore.Peers.Scheduler, as: Scheduler
```
Comment (d-velev) [inline apps/aehttpserver/web/controllers/info_controller.ex]: we don't need this

**Comment by cytadela8** [comment]
I just checked and `Module attributes are evaluated at compile time, not runtime.`, that makes this solution invalid.

File: apps/aehttpserver/web/controllers/info_controller.ex (hunk 3 above)
Comment (gspasov) [reply apps/aehttpserver/web/controllers/info_controller.ex]: We have missed those. A quick fix is on its way.

**Pull request closed without merging**